src/quo/components/share/share_qr_code/view.cljs

```diff
@@ -126,39 +128,42 @@
   [{:keys [share-qr-type qr-image-uri component-width customization-color full-name
            profile-picture emoji on-share-press]
     :as   props}]
-  [rn/view {:style style/content-container}
-   [rn/view
-    {:style style/share-qr-container}
+  [linear-gradient/linear-gradient
+   {:colors [(colors/resolve-color customization-color quo.theme/with-theme 15) :transparent]
```

Comment:
The view component needs to be wrapped with quo.theme and and pass the `theme` prop to `colors/resolve-color`. You can see example here: https://github.com/status-im/status-mobile/blob/develop/src/quo/components/wallet/network_routing/view.cljs#L189C1-L189C1

Reply:
I have updated to the code to wrap the outer view with `quo.theme`. Let me know if this is the preferred approach, or if we want to wrap the inner component?

Reply:
Looks good as is 👌